docs/source/mobile_optimizer.rst

```diff
@@ -0,0 +1,26 @@
+torch.utils.mobile_optimizer
+.. warning::
+
+This API is in beta and may change in the near future.
+
+Torch mobile supports :mod:`torch.mobile_optimizer.optimize_for_mobile` utility to run a list of optimization pass with modules in eval mode.
+The method takes the following parameters:
+
+1. A TorchScript object;
+2. A set of blacklisted optimization;
+3. A list of preserved methods.
+
+
+.. note::
+By default, if optimization blacklist is None or empty, “optimize_for_mobile” will run the following optimizations:
+
+1. MobileOptimizerType::CONV_BN_FUSION:  This optimization pass folds Conv2d-BatchNorm2d into Conv2d in forward method of this module and all its submodules. The weight and bias of the Conv2d are correspondingly updated.
+2. MobileOptimizerType::INSERT_FOLD_PREPACK_OPS: This optimization pass rewrites the graph to replace 2D convolutions and linear ops with their XNNPACK counterpart. XNNPACK is an open source library with kernels optimized for mobile platforms (such as arm). XNNPACK ops are stateful, that is they maintain a state per instance of the op which represent pre-packing of weights corresponding to the op. Weight pre-packing enables efficient memory access and thus faster kernel execution. optimize_for_mobile pass rewrites the graph to replace Conv2D/Linear with 1) op that pre-packs weight and 2) op that takes pre-packed weight and activation as input and generates output activations. Since 1 needs to be done only once, we fold the weight pre-packing such that it is done only once at model load time. This pass of the optimize_for_mobile does 1 and 2 and then folds weight pre-packing ops.
+3. ReLU/Hardtanh fusion: XNNPACK ops support fusion of clamping. That is clamping of output activation is done as part of the kernel, including for 2D convolution and linear op kernels. Thus clamping effectively comes for free. Thus any op that can be expressed as clamping op, such as ReLU or hardtanh, can be fused with previous Conv2D or linear op in XNNPACK. This pass rewrites graph by finding ReLU/hardtanh ops that follow XNNPACK Conv2D/linear ops, written by the previous pass, and fuses them together.
+4. MobileOptimizerType::REMOVE_DROPOUT: This optimization pass removes dropout and dropout_ nodes from this module when training is false.
+
+
+“optimize_for_mobile” will also invoke freeze_module pass which only preserve only forward method. If you have other method to that needed to be preserved,  add them into the preserved method list and pass into the method.
```

Comment:
Suggestion: Use either italics or code syntax for optimize_for_mobile of quotation marks.